internal/civisibility/utils/net/http.go

```diff
@@ -141,7 +141,7 @@ func (rh *RequestHandler) internalSendRequest(config *RequestConfig, attempt int
 		if log.DebugEnabled() {
 			var files []string
 			for _, f := range config.Files {
-				files = append(files, f.FileName)
+				files = append(files, f.FieldName)
```

Comment:
Is it really the field name that we need here?

(Sorry for leaving multiple individual comments, next time I will remember to group then into a proper review :sweat_smile:)

Reply:
this is only for the debug log entry: We don't store anything in the `FileName` field because is not required by the backend, so in order to have something meaningful in the debug logs I've changed it to the fieldname.